src/test/java/com/google/sps/ResponseTest.java

```diff
@@ -35,13 +37,13 @@ public void NoRerolls() {
     // A Response with only one restaurant.
     // pick() is called once, which should update the response status to NO_REROLLS.
 
-    List<Restaurant> restaurants = new ArrayList<>();
-    restaurants.add(new Restaurant("McDonald's", 4.5, "OPERATIONAL", new Photo[]{}));
-    Response response = new Response("OK", restaurants);
-    response.pick();
+    List<Restaurant> restArr = new ArrayList<>();
+    restArr.add(new Restaurant("McDonald's", 4.5, "OPERATIONAL", 2, new Photo[]{}));
+    Response response = new Response("OK", restArr);
+    RestaurantChooser.chooseRestaurant(response, 2);
 
     String actual = response.getStatus();
-    String expected = "NO_REROLLS";
-    assertEquals(expected, actual);
+    String expected = "ZERO_RESULTS";
+    Assert.assertEquals(expected, actual);
```

Comment:
I think the assertEquals was imported so you don't need to add Assert

Reply:
done